frontend/src/components/NotificationBox/NotificationBox.js

```diff
@@ -23,45 +26,46 @@ function NotificationBox(props) {
         </Alert>
       ));
 
+    const [notifications, setNotifications] = useState([]);
+
+    useEffect(() => {
+        getNotifications();
```

Comment:
You can place the function directly inside the useEffect hook to make it less redundant